ocr_service/main.py

```
@@ -0,0 +1,55 @@
+    description="Medical document OCR and data extraction service",
+    version="1.0.0",
+    docs_url="/docs",
+    redoc_url="/redoc",
+)
+
+# Include routers with API prefix
+app.include_router(ocr_router, prefix=settings.API_PREFIX)
+app.include_router(extraction_router, prefix=settings.API_PREFIX)
+
+@app.get("/")
+async def root():
```

Ruff (0.8.2)

Missing return type annotation for public function `root`

(ANN201)